tests/modules/nf-core/cellranger/count/nextflow.config

```diff
@@ -25,7 +25,7 @@ process {
     }
 
     withName: CELLRANGER_COUNT {
-        ext.args = '--chemistry SC3Pv3'
+        ext.args = '--chemistry SC3Pv3  --description sample-123'
```

Comment:
```suggestion
        ext.args = '--chemistry SC3Pv3'
```

Reply:
I did not see the param you proposed in the cellranger docs:
https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/count

Reply:
It's documented in the command-line help system.

```
$ cellranger count --help
cellranger-count
Count gene expression (targeted or whole-transcriptome) and/or feature barcode reads from a single sample and GEM well

USAGE:
    cellranger count [OPTIONS] --id <ID> --transcriptome <PATH>

OPTIONS:
        --id <ID>                                     A unique run id and output folder name [a-zA-Z0-9_-]+
        --description <TEXT>                          Sample description to embed in output files [default: ]
```

Reply:
For context this returns a string in the web_summary.html file so the reason I implemented this was to:

a. match results between Cell Ranger and UniverSC tests with 10X Genomics data.
b. enable unit testing with a "contain" check (which passes locally but was removed during review of GitHub Actions results)